C/hal/Ultrasonic/ultrasonic.h

```diff
@@ -0,0 +1,15 @@
+#ifndef ULTRASONIC_H_
+#define ULTRASONIC_H_
+
+typedef struct ultrasonic
+{
+    port_t port;
+    pin_t triggerPin;
+    pin_t echoPin;
+}Ultrasonic_t;
+
+error_t UltrasonicInit(Ultrasonic_t * pUS, uint8_t kMode,
+                        uint8_t kClock, uint8_t kInterruptSource);
+error_t UltrasonicTrigger(Ultrasonic_t * pUS);
```

Comment:
For consistency we should name those functions to be like 
`Ultrasonic_Init()` `Ultrasonic_Trigger()`